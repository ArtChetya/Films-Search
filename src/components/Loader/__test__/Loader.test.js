import { shallow } from 'enzyme/build';
import React from 'react';
import { Loader } from '..';

describe('Loader component', () => {
    it('should be rendered correctly without props', () => {
        const component = shallow(<Loader />);
        expect(component).toMatchSnapshot();
    });
});
