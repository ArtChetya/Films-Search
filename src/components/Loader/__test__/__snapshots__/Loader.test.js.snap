// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should be rendered correctly without props 1`] = `
<Loader__StyledGrid
  alignItems="center"
  container={true}
  justify="center"
>
  <Loader__Progress
    loadercolor="#ec407a"
    size="90px"
  />
</Loader__StyledGrid>
`;
